replacement.json

```diff
@@ -0,0 +1,106 @@
+{
+	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
+	"all": {
+		"extends": [
+			"github>TWiStErRob/renovate-config:replacement/gradle-build-action-to-setup-gradle",
+			"github>TWiStErRob/renovate-config:replacement/jcenter-flexbox-to-gmaven-flexbox",
+			"github>TWiStErRob/renovate-config:replacement/nhaarman-mockito-kotlin-v1-to-v2",
+			"github>TWiStErRob/renovate-config:replacement/nhaarman-mockitokotlin2-to-mockito-kotlin",
+			"github>TWiStErRob/renovate-config:replacement/gmazzo-gradle-buildconfig-plugin-v3-to-v4",
+			"github>TWiStErRob/renovate-config:replacement/gradle-enterprise-develocity-plugin",
+			"github>TWiStErRob/renovate-config:replacement/squareup-retrofit-v1-to-v2",
+			"github>TWiStErRob/renovate-config:replacement/squareup-okhttp-v2-to-v3"
```

Comment:
Not supported: https://github.com/renovatebot/renovate/discussions/24883#discussioncomment-8388613

```gradle
// build.gradle.kts
plugins {
	id("com.android.application") version "8.3.1"
	kotlin("android") version "1.9.23"
}

dependencies {
	implementation("com.google.android:flexbox:2.0.1")
	implementation("com.github.gmazzo:gradle-buildconfig-plugin:3.1.0")
}
android {
	namespace = "com.example"
	compileSdk = 34
	defaultConfig {
		minSdk = 21
		targetSdk = 34
	}
}
```

```gradle
// settings.gradle.kts
pluginManagement {
	repositories {
		google()
		mavenCentral()
		gradlePluginPortal()
		jcenter()
	}
}

plugins {
	id("com.gradle.enterprise") version "3.17"
}

@Suppress("UnstableApiUsage")
dependencyResolutionManagement {
	repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
	repositories {
		google()
		mavenCentral()
		gradlePluginPortal()
		jcenter()
	}
}
```

```toml
# libs.versions.toml
[versions]
flexbox = "2.0.1"

[libraries]
flexbox = { module = "com.google.android:flexbox", version.ref = "flexbox" }
```
![image](https://github.com/TWiStErRob/renovate-config/assets/2906988/aaa6b821-3aa6-4cea-abd4-bb1163b6f4bb)